models/ldndc/tests/testthat/test.met2model.R

```diff
@@ -0,0 +1,18 @@
+context("met2model")
+
+outfolder <- tempfile()
+setup(dir.create(outfolder, showWarnings = FALSE))
+teardown(unlink(outfolder, recursive = TRUE))
+
+test_that("Met conversion runs without error", {
+  skip("This is a template test that will not run. To run it, remove this `skip` call.")
+  nc_path <- system.file("test-data", "CRUNCEP.2000.nc",
+                         package = "PEcAn.utils")
+  in.path <- dirname(nc_path)
+  in.prefix <- "CRUNCEP"
+  start_date <- "2000-01-01"
+  end_date <- "2000-12-31"
+  result <- met2model.MODEL(in.path, in.prefix, outfolder, start_date, end_date)
```

Comment:
this should be:
```suggestion
  result <- met2model.LDNDC(in.path, in.prefix, outfolder, start_date, end_date)
```
you could also write different test